examples/meta/src/meta_api/enums.sg

```diff
@@ -0,0 +1 @@
+enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC
```

Comment:
A third option and (probably the cleanest) is to try to get enum typed variables working. I.e. get the meta-language statement `LIBLINEAR_SOLVER_TYPE myVar = enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC` to work. I actually don't think this would be very difficult - these types of statements already parse and translate. You just need to make sure that enum types like `LIBLINEAR_SOLVER_TYPE` are imported correctly in all the languages. That's just a matter of fixing the dependency translation rules in the target `.json` files. Try updating your example and see what travis says. Let me know if you get stuck.